cogs/Admin.py

```diff
@@ -0,0 +1,154 @@
+import discord
+from discord.ext import commands
+from textwrap import dedent
+from common import schemas
+from common import schemas
+from textwrap import dedent
+import logging
+from royale_api_website_scraper import RoyaleApiWebsiteScraper
+from clan_members_rank import ClanMembersRanker
+
+def create_clan_members_ranked_embed(clan_info, clan_members_ranked, card_type_arg='all'):
+    n = 20  # Number of players to show
+    top_n = clan_members_ranked[:n]
+
+    embed = discord.Embed(
+        description=dedent('''
+            Players ranked by number of cards they have at each level. Comparison start at level 14 card count.
+            Showing the top **{}** players.
+            Card Count Filter Type: **{}**
+        '''.format(n, card_type_arg)),
+        colour=discord.Colour.blue()
+    )
+
+    embed.set_author(
+        name=clan_info['name'],
+        icon_url='https://icon-library.net//images/clash-royale-icon/clash-royale-icon-8.jpg'
+    )
+
+    rank_values = []
+    for idx, member in enumerate(top_n):
+        mc = member['card_level_counts']
+        rank_text_row = '` {:2d} ` | ` ({:2d}, {:2d}), {:2d}, {:2d} ` | **{}**'.format(
+            idx + 1, mc[14], mc[13], mc[12], mc[11], member['name']
+        )
+        rank_values.append(rank_text_row)
+    field_name = '**Rank** | **# of lvl (14, 13), 12, 11** | **Player Name**'
+    embed.add_field(name=field_name, value='\n'.join(rank_values), inline=False)
+
+    return embed
+
+def boat_attackers_embed(boat_attackers, clan_info):
+    if len(boat_attackers) == 0:
+        description = 'There are no clan members who attacked enemy boats this week.'
+        colour = discord.Colour.blue()
+    else:
+        description = f'**{len(boat_attackers)}** player(s) attacked enemy boats this week.'
+        colour = discord.Colour.orange()
+
+    embed = discord.Embed(description=description, colour=colour)
+    embed.set_author(name=clan_info['name'], icon_url=clan_info['logo_url'])
+
+    if len(boat_attackers) != 0:
+        columns = 'Boat attacks | Medals | Name'
+        row_values = []
+        for p in boat_attackers:
+            row_val = '`     {:^2}    `|`  {:^4} `| {}'.format(
+                p[5], p[6], p[1]
+            )
+            row_values.append(row_val)
+        embed.add_field(name=columns, value='\n'.join(row_values), inline=False)
+
+    return embed
+
+class Admin(commands.Cog):

```

Comment:
What's the suggested naming convention of COG classes? Should we give it a better name rather than "Admin"?

Reply:
created a basic_command.py cog file and a clash_river_war cog file